libfreerdp/codec/xcrush.c

```diff
@@ -734,11 +734,24 @@ static int xcrush_generate_output(XCRUSH_CONTEXT* xcrush, BYTE* OutputBuffer, UI
 
 static size_t xcrush_copy_bytes(BYTE* dst, const BYTE* src, size_t num)
```

Comment:
Declaring this function `INLINE` might also help

Reply:
Yes